components/scripts/create.tsx

```diff
@@ -17,22 +17,22 @@ export default function Create() {
     const [loading, setLoading] = React.useState(false);
 
     const handleSubmit = async () => {
-        getNewScriptName()
-            .then((name) => {
-                createScript({
-                    displayName: name,
-                    visibility: "private",
-                    content: newDefaultAssistant(name),
-                }).then((script) => {
-                    getScript(`${script.id}`)
-                        .then((script) =>
-                            window.location.href = `/edit?file=${script?.publicURL}&id=${script?.id}`
-                        );
-                })
-            })
+        const defaultName = "New Assistant";
+        const slug = defaultName.toLowerCase().replace(" ", "-") + "-" + Math.random().toString(36).substring(2, 7);
```

Comment:
The logic of creating a unique slug per assistant by appending random 5 charaters.